frontend/Dockerfile

```
@@ -0,0 +1,15 @@
+FROM damnit-web-site:test AS site
+FROM damnit-web-app:test AS app
+
+FROM nginx:stable-alpine AS frontend
+
+COPY nginx/templates/default.frontend.conf.template /etc/nginx/templates/default.conf.template
+COPY nginx/proxy_api.conf /etc/nginx/proxy_api.conf
+
+COPY --from=site /usr/src/app/apps/site/dist /usr/share/nginx/html/site
+COPY --from=app /usr/src/app/apps/app/dist  /usr/share/nginx/html/app
+
+HEALTHCHECK --interval=30s --timeout=3s \
+  CMD wget -qO- http://127.0.0.1/ || exit 1
```

I think with our nginx configs in `frontend/nginx/templates` we're only listening to `443`, not `80`. Isn't really a visible issue when loading the page since a browser will do the redirect for you.

---

IMO we should update the nginx config to auto-redirect to `443` for clarity, and this check should be done one https:

```suggestion
HEALTHCHECK --interval=30s --timeout=3s \
CMD wget -qO- --no-check-certificate https://127.0.0.1/ || exit 1
```

That way the healthcheck fails if we're not serving over 443.